interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Controller/EncounterccdadispatchController.php

```diff
@@ -103,8 +103,12 @@ public function indexAction()
             $view = $this->params('view');
             $pids = explode('|', $combination);
             $measures = $_REQUEST['report_measures'] ?? "";
-            if (is_array($measures) && empty($measures[0])) {
-                $measures = ''; // defaults to all current.
+            if (is_array($measures)) {
+                if (empty($measures[0])) {
+                    $measures = ''; // defaults to all current one per patient.
+                } elseif (!empty($measures[0] == 'all')) {
```

Comment:
I sometimes wonder what the heck I was thinking the next day!
!empty($measures[0] == 'all')